src/ui/actions/logpoint.ts

```diff
@@ -276,7 +281,7 @@ async function setMultiSourceLogpoint(
     analysis = await createAnalysis(params);
     const { analysisId } = analysis;
 
-    store.dispatch(analysisCreated({ analysisId, location: locations[0], condition }));
+    store.dispatch(analysisCreated({ analysisId, location: locations[0], condition, focusRegion }));
```

Comment:
What do you think about explicitly passing `params.range` here? I think for the purposes of caching in the future, we will not care about it being a `focusRegion` in particular, and instead will want a `TimeStampedPointRange`, since that is a more generic type.

Reply:
This might also transition more smoothly to when/if we eventually re-add the concept of zoom region in the UI. Then the "range" could just be focus (if there is one) or zoom (if there is one) or null.